wzdx/tools/cdot_geospatial_api.py

```diff
@@ -5,8 +5,8 @@
 
 from ..tools import geospatial_tools, path_history_compression
 
-# BASE_URL = "https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded"
-BASE_URL = "https://dtdapps.colorado.gov/server/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded"
+BASE_URL = "https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded"
```

Comment:
Should this be abstracted to environment variable so we can more easily compare between the two as needed?

Reply:
Updated this to to be defined within the class, and defined by the init param, then by env var, then by the default str.